$NetBSD: patch-ak,v 1.2 2010/09/02 06:58:35 obache Exp $

gpg-zip script will not be installed, so man page also should not be installed.
If will introduce conflict with security/gnupg.

--- doc/Makefile.am.orig	2009-12-16 18:23:55.000000000 +0000
+++ doc/Makefile.am
@@ -67,7 +67,7 @@ myman_sources = gnupg7.texi gpg.texi gpg
 myman_pages   = gpg2.1 gpgsm.1 gpg-agent.1 scdaemon.1 gpgv2.1 \
                 watchgnupg.1 gpgconf.1 addgnupghome.8 gpg-preset-passphrase.1 \
 		gpg-connect-agent.1 gpgparsemail.1 symcryptrun.1 \
-		gpgsm-gencert.sh.1 applygnupgdefaults.8 gpg-zip.1
+		gpgsm-gencert.sh.1 applygnupgdefaults.8
 
 man_MANS = $(myman_pages)
 noinst_MANS = gnupg.7
